### PR TITLE
Delete confirm enhanced

### PR DESCRIPTION
**Pull request by laurieboyes**
Sorry, it got a bit big

Delete confirmation and enhanced journey

Core
screen shot 2015-12-08 at 12 29 24

Enhanced
screen shot 2015-12-08 at 12 29 37

Result
screen shot 2015-12-08 at 12 29 51

### CODE REVIEW

File: client/components/lists/main.scss [hunk 1]
```diff
@@ -151,3 +155,8 @@
 	}
 
 }
+
+// HACK: don't understand why this is necessary to stop the row leaking 20px left out of the container
```
Comment (laurieboyes) [inline client/components/lists/main.scss]: If anyone has any o-grid expertise they think might be able to help me I'll be forever grateful

File: views/cards.html [hunk 2]
```diff
@@ -9,6 +9,14 @@
...
+				<div class="o-grid-row myft__dashboard__message-row-padding-hack">
+					<div class="o-forms-message o-forms-message--success myft-list-form__message"
+						 data-o-grid-colspan="12">
+						<p>You have successfully deleted the list</p>
+					</div>
+				</div>
+			{{/ifEquals}}
 			<div class="o-grid-row">
 				{{#each cards}}
 					<div class="myft__dashboard-card">
```
Comment (ironsidevsquincy) [inline views/cards.html]: re: padding, maybe this needs a `data-o-grid-colspan`?

**Comment by keirog** [comment]
Tiny arrow heading could do with non-tinifying but otherwise :+1:

File: views/cards.html (hunk 2 above)
Comment (laurieboyes) [inline views/cards.html]: cheers darren, I worked out eventually that there wasn't actually any need for me to stick it in an o-row anyway

**Pull request merged**